src/server/pps/server/api_server.go

```diff
@@ -3135,6 +3136,83 @@ func (a *apiServer) ActivateAuth(ctx context.Context, req *pps.ActivateAuthReque
 	return &pps.ActivateAuthResponse{}, nil
 }
 
+// RunLoadTestDefault implements the pps.RunLoadTestDefault RPC
+// TODO: It could be useful to make the glob and number of pipelines configurable.
+func (a *apiServer) RunLoadTestDefault(ctx context.Context, _ *types.Empty) (_ *pfs.RunLoadTestResponse, retErr error) {
+	func() { a.Log(nil, nil, nil, 0) }()
+	defer func(start time.Time) { a.Log(nil, nil, retErr, time.Since(start)) }(time.Now())
+	pachClient := a.env.GetPachClient(ctx)
+	repo := "load_test"
+	if err := pachClient.CreateRepo(repo); err != nil && !pfsServer.IsRepoExistsErr(err) {
+		return nil, err
+	}
+	branch := uuid.New()
+	if err := pachClient.CreateBranch(repo, branch, "", "", nil); err != nil {
+		return nil, err
+	}
+	pipeline := tu.UniqueString("TestLoadPipeline")
```

Comment:
Should we incorporate the time into this somehow, maybe in the description for the pipeline?  If there will only ever be one in a cluster then this is fine.  But if we are reusing a cluster we should have them timestamped.

Reply:
Yeah, there are definitely some changes like this that we should make over time. The main goal with this is an MVP for default tests that we can periodically run in Hub.